ambry-api/src/main/java/com.github.ambry/config/ReplicationConfig.java

```diff
@@ -106,7 +106,7 @@ public ReplicationConfig(VerifiableProperties verifiableProperties) {
     replicationTokenFlushDelaySeconds =
         verifiableProperties.getIntInRange("replication.token.flush.delay.seconds", 5, 1, Integer.MAX_VALUE);
     replicationFetchSizeInBytes =
-        verifiableProperties.getLongInRange("replication.fetch.size.in.bytes", 1048576, 0, 20971520);
+        verifiableProperties.getLongInRange("replication.fetch.size.in.bytes", 1048576, 1, Long.MAX_VALUE);
```

Comment:
MAX_VALUE does not make sense for such params. Would prefer not using a range method for this one really.

Reply:
This also serves as a "not negative" check - for that this might be useful, no?

Reply:
Yeah, I think we have gone over this before. I don't think that is very meaningful. Long.MAX_VALUE is as wrong as -1 for this param. Not having a range is probably more sensible. Ideally, we should perhaps have separate methods for min bounds and max bounds.

Reply:
+1 on the different method. We have enough use cases to do it. I'll create an issue for it

Reply:
https://github.com/linkedin/ambry/issues/886